src/main/java/com/ericsson/ei/subscriptionhandler/RunSubscription.java

```diff
@@ -44,8 +46,11 @@
     private JmesPathInterface jmespath;
 
     static Logger log = (Logger) LoggerFactory.getLogger(RunSubscription.class);
```

Comment:
Change here and in other files you changed to private static final Logger LOGGER